### DISCUSSION TITLE
After enabling a module it doesnt create overlay variables

### DISCUSSION THREAD

**Discussion by OmniversePhotography** [AllskyTeam/allsky-modules#81] [Q&A]
I am having some issues with modules since updating to Bookworm. After getting the dew heater module installed, i set its parameters, enabled it, and save it in the module manager. According to the documentation page on modules they are supposed to create variables in the variable manager on the overlay screen. This has been my experience in the past with this very same module, but this time around it is not creating the variables automatically.

**Comment by OmniversePhotography** [comment]
Never mind, I left it running and after while it showed up. I thought it was immediate.

**Comment by Alex-developer** [comment]
The variables will only be available after the module has first run. So after adding it to the flow allow it to run then look in the variable manager

**Reply by OmniversePhotography** [reply]
good to know for the future, thanks!
